tools/cldr-apps/.settings/org.eclipse.wst.common.component

```diff
@@ -1,78 +1,81 @@
 <?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
-                            
+                                
+    
     
     
     
     
     
     <wb-module deploy-name="cldr-apps">
-                                
+                                        
+        
         
         
         <wb-resource deploy-path="/" source-path="/target/m2e-wtp/web-resources"/>
-                                                        
+                                                                
+        
         
         
         
         
         
         <wb-resource deploy-path="/" source-path="/src/main/webapp" tag="defaultRootSource"/>
-                                                        
+                                                                
+        
         
         
         
         
         
         <wb-resource deploy-path="/WEB-INF/lib" source-path="/WebContent/WEB-INF/lib"/>
-                                                 
+                                                         
+        
         
         
         
         
         
         <!-- needed? -->
-                                                        
+                                                                
+        
         
         
         
         
         
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/.apt_generated"/>
-                                        
+                                                
+        
         
         
         
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/resources"/>
-                
-        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/java"/>
-                
-        <dependent-module archiveName="cldr-code-39.0-SNAPSHOT.jar" deploy-path="/WEB-INF/lib" handle="module:/resource/cldr-code/cldr-code">
-                        
-            <dependency-type>uses</dependency-type>
-                    
-        </dependent-module>
-                
-        <dependent-module archiveName="cldr-rdf-39.0-SNAPSHOT.jar" deploy-path="/WEB-INF/lib" handle="module:/resource/cldr-rdf/cldr-rdf">
                         
+        
+        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/java"/>
+        <dependent-module archiveName="cldr-code-40.0-SNAPSHOT.jar" deploy-path="/WEB-INF/lib" handle="module:/resource/cldr-code/cldr-code">
```

Comment:
the change from 39 to 40 might be the only significant change; the rest looks like whitespace

I'm not very sure when such changes should be committed for this file -- but we're certainly not on 39 anymore